torchaudio/compliance/kaldi.py

```diff
@@ -779,7 +787,9 @@ def _get_num_LR_output_samples(input_num_samp, samp_rate_in, samp_rate_out):
     return num_output_samp
 
 
+@torch.jit.export
 def resample_waveform(waveform, orig_freq, new_freq, lowpass_filter_width=6):
```

Comment:
We no longer compile functionals and transforms by default. Therefore, we no longer add annotations. However, we do test that the [functionals](https://github.com/pytorch/audio/blob/master/test/test_functional.py#L20) and [transforms](https://github.com/pytorch/audio/blob/master/test/test_transforms.py#L23) can be compiled.